x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -367,7 +367,7 @@ function jobProperty(job, prop) {
   return job[propMap[prop]];
 }
 
-export function getJobIdUrl(jobId) {
+export function getJobIdUrl(tabId, jobId) {
```

Comment:
This could be moved to a common directory since it's shared (maybe `public/application/util`?).

Could just create a new ts file with the name of the function with only that function in that file (like  `job_selection_change.ts`, `has_matching_points.ts`). Then you wouldn't have to worry about `jobs_list/components/utils.js` not being in ts yet.

Reply:
I moved this `getJobIdUrl` function to `x-pack/plugins/ml/public/application/util/get_job_id_url.ts` in https://github.com/elastic/kibana/pull/65778/commits/f3c8367a68b4abffa1dff9f3b9b7c456ecc1552a. Much cleaner 😄 !